tests/e2e/Services/Sites/SitesCustomServerTest.php

```diff
@@ -1010,41 +1010,6 @@ public function testGetDeployment(): void
         $this->cleanupSite($siteId);
     }
 
-    // public function testLoadSite(): void
```

Comment:
leftovers

Reply:
It was leftover, this is removing it. it was just piece of comment as "template" for tests, before they were implemented. Now we already got dozens of tests with sites, including flow with router. This was just a leftover

Reply:
We use ours only if site is static, and site is not SPA